api/internal/localizer/localizer.go

```diff
@@ -75,9 +76,51 @@ func (lc *Localizer) Localize() error {
 	return nil
 }
 
-// TODO(annasong): implement
-// processKust returns a copy of the kustomization at kt with paths localized.
-func (lc *Localizer) processKust(kt *target.KustTarget) *types.Kustomization {
+// processKust returns a copy of the kustomization at kt with the patches field localized
+func (lc *Localizer) processKust(kt *target.KustTarget) (*types.Kustomization, error) {
 	kust := kt.Kustomization()
-	return &kust
+	for name, patches := range map[string][]types.Patch{
+		"patches":         kust.Patches,
+		"patchesJson6902": kust.PatchesJson6902,
+	} {
+		for i := range patches {
+			if patches[i].Path != "" {
+				newPath, err := lc.localizeFile(patches[i].Path)
+				if err != nil {
+					return nil, errors.WrapPrefixf(err, "unable to localize %s path", name)
```

Comment:
```suggestion
					return nil, errors.WrapPrefixf(err, "unable to localize path %q", name)
```

Reply:
Sorry this was confusing. `name` is the name of the field being localized, i.e. `patches` and `patchesJson6902`, not the actual path. 

I changed `name` to `fieldName`. Hope this helps.

Reply:
Done.